chapters/sampling-randomization-power.tex

```diff
@@ -1,529 +1,782 @@
+Your data map must indicate which ID variables can be used and how to merge datasets.
+It is common that administrative data use IDs 
+that are different than the project IDs, 
+and the linkage between those should be clear from your master dataset.
+
+The data map should also include metadata about the handling of all information.
+These characteristics may be updated as the project progresses.
+For example, you will need to note the original source of each dataset,
+as well as the project folder where 
+the raw original data and codebooks are stored
+and where the back-ups for the each raw dataset are stored.
+
+Some of the characteristics in your master datasets and your data map
+should be filled in during the planning stage,
+but both of them should be active resources 
+that are updated as your project evolves.
+Finally, your master data should not include any unlabeled missing values. 
+If the information is missing for one unit, 
+then the reason should always be indicated with a code.
+An example for such reason could be that a unit was not included in the treatment assignment
+as it was not sampled in the first place,
+or was not located in the data collection at a given round.
+
+\subsection{Defining study comparisons using master data}
+
+Your research design will determine what statistical comparisons you need
+to estimate in your analytical work.
+The research designs discussed here compare a group that received
+some kind of \textbf{treatment}\sidenote{
+	\textbf{Treatment:} The general word for the evaluated intervention or event.
+	This includes being offered training or cash transfer from a program, experiencing a natural disaster etc.}
+against a counterfactual control group.\sidenote{
+	\textbf{Counterfactual:} A statistical description of what would have happened
+	to specific individuals in an alternative scenario,
+	for example, a different treatment assignment outcome.}
+\index{counterfactual}
+The key assumption is that each
+person, facility, or village (or whatever the unit of intervention is)
+had two possible states: their outcome if they did receive the treatment
+and their outcome if they did not receive that treatment.
+The average impact of the treatment is defined as
+the difference between these two states averaged over all units.
+
+However, we can never observe the same unit
+in both the treated and untreated state simultaneously,
+so we cannot calculate these differences directly.
+Instead, the treated group is compared to a control group
+that is statistically indistinguishable,
+which makes the average impact of the treatment
+mathematically equivalent to the difference in averages between the groups.
+Statistical similarity is often defined
+as \textbf{balance} between two or more groups.	
+This test is so common,	
+DIME Analytics created the Stata command \texttt{iebaltab}\sidenote{	
+	\url{https://dimewiki.worldbank.org/iebaltab}}	
+to generates tables of balance checks	
```

Comment:
would be useful to add 1 sentence on what iebaltab does and what makes it unique

Reply:
imagine there will be the same objection to this as to iematch, but i think it's useful to add a little more on what iebaltab does well (automates a process done commonly, creates nicely formatted outputs)

Reply:
Suggested rephrase: 
Since balance tests are commonly run for impact evaluations, DIME Analytics created a Stata command to standardize and automate the creation of nicely-formatted balance tables: \texttt{iebaltab}\sidenote{	
	\url{https://dimewiki.worldbank.org/iebaltab}}.

Reply:
Also sounds good to me, done in f67135387749abad7273236c10bf0b7f3ca56dfc